scripts/init.sh

```diff
@@ -136,7 +136,7 @@ start-frequency-instant)
   ;;
 
 start-frequency-interval)
-  defaultInterval=12
+  defaultInterval=6
```

Comment:
Do we want to also change the default for the `sealing_interval` parameter? (doesn't have to be this PR)

Reply:
Good idea! I update the default to 6 seconds.
```
#[clap(long, help = "The interval in seconds", default_value = "6", value_name = "SECONDS")]
```